src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -596,18 +584,36 @@ private function getOrderDocuments(Order $order): OrderDocumentsForViewing
                         );
                     }
                 }
-            } elseif ($document instanceof OrderSlip) {
+            } elseif (OrderDocumentType::DELIVERY_SLIP === $type) {
+                $number = sprintf(
+                    '%s%06d',
+                    Configuration::get('PS_DELIVERY_PREFIX', $this->contextLanguageId, null, $order->id_shop),
+                    $document->delivery_number
+                );
+                $amount = $this->locale->formatPrice(
+                    $document->total_shipping_tax_incl,
+                    $currency->iso_code
+                );
+                $numericAmount = $document->total_shipping_tax_incl;
+            } elseif (OrderDocumentType::CREDIT_SLIP) {
+                $number = sprintf(
+                    '%s%06d',
+                    Configuration::get('PS_CREDIT_SLIP_PREFIX', $this->contextLanguageId),
+                    $document->id
+                );
                 $amount = $this->locale->formatPrice(
                     $document->total_products_tax_incl + $document->total_shipping_tax_incl,
                     $currency->iso_code
                 );
+                $numericAmount = $document->total_products_tax_incl + $document->total_shipping_tax_incl;
```

Comment:
Just curious: was the numericalAmount wrong in some cases before this modification? 🤔

Reply:
got my answer 👍

Reply:
Yes it was missing for some documents, and inconsistent For example the formatted value was the sum of  `$document->total_products_tax_incl + $document->total_shipping_tax_incl` but the numeric amount was just `$document->total_products_tax_incl`